### ISSUE TITLE
Resources cost

### ISSUE THREAD

**Issue by mahmoudaljan** [timnon/pyschedule#55]
Is there a way to set a cost per unit for resources such as the solver will minimize the execution cost of a schedule?

**Comment by timnon** [comment]
No, but it would be relatively easy to add. There is some `schedule_cost` for each task which gives some cost/reward in case a task is scheduled. I was thinking about doing sth similar for resources such that there is some cost a resource is used. But you want a cost per time slot used on a resource?

**Comment by mahmoudaljan** [comment]
@timnon Yes, the goal is to set a cost per time unit per resource ( employee A costs 15$ per hour, employee B costs 30$ per hour) and optimize for the best overall cost of execution as opposed to makespan

**Comment by mahmoudaljan** [comment]
@timnon it seems the schedule_cost feature for each task approach doesnt help. Any lead on how to implement such a requirement ?

**Comment by timnon** [comment]
Hi, just added an option 'cost_per_period' to any resource, e.g. `R = S.Resource('R`,cost_per_period=15)`

Please download the newest version 0.2.24 and test

**Comment by perswedenborg** [comment]
It would be great resources also could have different costs for different time periods, e.g. employee “A” costs more if a task is assigned to him during a weekend/evening vs normal work hours.
